app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
```

Comment:
Might want to validate uniqueness of username and email to prevent two or more users having the same email or username

[OPTIONAL]